recipes/libfranka/recipe.yaml

```diff
@@ -0,0 +1,63 @@
+context:
+  name: libfranka
+  version: "0.9.2"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  - url: https://github.com/frankarobotics/libfranka/archive/refs/tags/0.9.2.tar.gz
+    sha256: 5ddce13810c31b6365c43b22a08c84f4f8ad1c4d778701586e37f8810902a92a
+
+  # Exact commit hash was obtained from the submodule inside the target libfranka version
+  - url: https://github.com/frankarobotics/libfranka-common/archive/e6aa0fc210d93fe618bfd8956829a264d5476ba8.tar.gz
+    sha256: 721fffe11c5dec6735e191c2f08e58de3828d05ea25c3901c1940cd30a226509
+    target_directory: common
+
+
+
+build:
+  skip:
+    - win
+    - osx
+  number: 0
+
+requirements:
+  build:
+    - ${{ compiler('cxx') }}
+    - ${{ stdlib('c') }}
+    - cmake
+    - make
+    - ninja
+  host:
+    - eigen
+    - poco <1.14
+    - fmt
+  run_exports:
+    - ${{ pin_subpackage('libfranka', exact=True) }}
```

Comment:
`exact` is too strict, as it would constraint also build number and hash. If the library does not make any ABI promise, I think it make sense to have `upper_bound='x.x.x'`